projects/1.16/assets/draconic-evolution/draconicevolution/lang/zh_cn.json

```
@@ -288,306 +297,311 @@
-  "module.draconicevolution.shield_control.name": "护盾冷却时间",
+  "module.draconicevolution.shield_control.name": "护盾冷却",
   "module.draconicevolution.shield_control.value": "%s秒",
-  "module.draconicevolution.shield_passive.name": "操作护盾消耗",
-  "module.draconicevolution.shield_passive.value": "%s OP/t",
-  "module.draconicevolution.shield_recharge.name": "护盾重充能",
-  "module.draconicevolution.shield_recharge.value": "%s 点每秒（共%s秒）",
+  "module.draconicevolution.shield_passive.name": "能量运行消耗",
+  "module.draconicevolution.shield_passive.value": "%sOP/t",
+  "module.draconicevolution.shield_recharge.name": "护盾恢复",
+  "module.draconicevolution.shield_recharge.value": "%s点/秒（%s秒\n消耗%sOP/t",
   "module.draconicevolution.speed.name": "速度",
   "module.draconicevolution.speed.value": "+%s%%",
   "module.draconicevolution.undying.charge.name": "充能时间",
-  "module.draconicevolution.undying.charge.value": "%s Seconds",
-  "module.draconicevolution.undying.energy.name": "Charge Energy",
+  "module.draconicevolution.undying.charge.value": "%s 秒",
+  "module.draconicevolution.undying.energy.name": "充能",
   "module.draconicevolution.undying.energy.value": "%sOP @%s OP/t",
-  "module.draconicevolution.undying.health.name": "Health Boost",
-  "module.draconicevolution.undying.health.value": "%s Health points",
-  "module.draconicevolution.undying.invuln.active": "Invulnerable for %s Seconds",
-  "module.draconicevolution.undying.invuln.name": "Invulnerable Time",
-  "module.draconicevolution.undying.invuln.value": "%s Seconds",
-  "module.draconicevolution.undying.shield.name": "Shield Boost",
-  "module.draconicevolution.undying.shield.value": "%s for %s seconds",
+  "module.draconicevolution.undying.health.name": "生命提升",
+  "module.draconicevolution.undying.health.value": "%s点生命值",
+  "module.draconicevolution.undying.invuln.active": "无敌%s秒",
+  "module.draconicevolution.undying.invuln.name": "无敌时间",
+  "module.draconicevolution.undying.invuln.value": "%s秒",
+  "module.draconicevolution.undying.shield.name": "护盾瞬间提升",
+  "module.draconicevolution.undying.shield.value":"%s秒内提升护盾%s点",
   "module_type.draconicevolution.aoe.name": "范围",
-  "module_type.draconicevolution.aqua_adapt.name": "Aqua Adaptation",
+  "module_type.draconicevolution.aqua_adapt.name": "水下适应",
   "module_type.draconicevolution.auto_feed.name": "自动进食",
   "module_type.draconicevolution.auto_fire.name": "自动射击",
   "module_type.draconicevolution.damage.name": "伤害",
-  "module_type.draconicevolution.energy_link.name": "Energy Link",
-  "module_type.draconicevolution.energy_share.name": "Energy Share",
-  "module_type.draconicevolution.energy_storage.name": "储能容量",
+  "module_type.draconicevolution.energy_link.name": "能量链接",
+  "module_type.draconicevolution.energy_share.name": "能源共享",
+  "module_type.draconicevolution.energy_storage.name": "能量容量",
   "module_type.draconicevolution.flight.name": "飞行",
   "module_type.draconicevolution.hill_step.name": "自动跳跃",
```

```suggestion
  "module_type.draconicevolution.hill_step.name": "自动上坡",
```
它和原版的自动跳跃还有点不一样，应该效果是直接让玩家能像走台阶一样走上一定高度的方块，或许应该作区分？

---

自动拾级？

---

名字叫hill step......
如果叫step assist的话可以叫*步行辅助*了（好像有些模组是这么说的）（

---

与原版效果应该是完全一样，能自动走上1格高的方块（1.5格上不去）

---

原版的“自动跳跃”是在接近方块时[自动进行了一次跳跃]一下以便登上1格的高低差
模组里的功能应该是[增高玩家可以踏上的高度]以便玩家能直接走上1格（或更高）的高低差（使用装备属性forge:step_height_addition可以实现类似效果）
得明确一下

---

其实自动上坡就挺符合原文的，自动拾级感觉不太准确啊